src/web-auth/captcha.js

```diff
@@ -114,30 +154,60 @@ function handleRecaptchaProvider(element, options, challenge) {
 
   if (widgetId) {
     setValue();
-    globalForRecaptchaProvider(challenge.provider).reset(widgetId);
+    if (challenge.provider !== FRIENDLY_CAPTCHA_PROVIDER) {
+      globalForCaptchaProvider(challenge.provider).reset(widgetId);
+    }
```

Comment:
Should we call reset on the generated widget instance when using friendly-captcha ?

Reply:
I attempted this but was unable to find a way to achieve this.

With the other providers, the `render` function returns the `id` of the element. We then add the attribute `'data-wid'`.

At the start of the captcha logic, we find the element with `data-wid`. We then use the global provider to reset that element, using the element as a parameter.

For Friendly Captcha, we cannot find the `id` of the created instance. We also need to call `reset` on the `WidgetInstance` itself and not from the global provider.

This is a great point that you bring up though. Let me brainstorm this with the team a little to make sure we've exhausted all options and this will not introduce unexpected behavior.

Reply:
@frederikprijck we're now storing the `WidgetInstance` on the `window` object for later use and had include `reset` logic. Thanks again.